packages/react-components/src/components/VideoGallery/utils/videoGalleryLayoutUtils.ts

```diff
@@ -96,8 +98,19 @@ const _useOrganizedParticipants = (props: OrganizedParticipantsArgs): OrganizedP
 
   const gridParticipants = getGridParticipants();
 
-  const getOverflowGalleryRemoteParticipants = useCallback((): VideoGalleryRemoteParticipant[] => {
-    if (isScreenShareActive) {
+  const getOverflowGalleryRemoteParticipants = useCallback((): (
+    | VideoGalleryParticipant
+    | VideoGalleryRemoteParticipant
+  )[] => {
+    if (isScreenShareActive && localParticipant) {
+      const localParticipantPlusOverflow = [localParticipant].concat(
+        visibleGridParticipants.current.concat(visibleOverflowGalleryParticipants.current)
+      );
+      // If screen sharing is active, assign video and audio participants as overflow gallery participants
+      /* @conditional-compile-remove(PSTN-calls) */ /* @conditional-compile-remove(one-to-n-calling) */
+      return localParticipantPlusOverflow.concat(callingParticipants);
+      return localParticipantPlusOverflow;
+    } else if (isScreenShareActive) {
```

Comment:
I don't want to make the ordering and organizing of remote participants more complicated. Is it possible to prepend the local video tile to the return overflow gallery participants of this hook?

Reply:
I found it easier to do the mapping in here than in the result of this hook since it is also returning the grid organization too